cypress/integration/crud.spec.js

```diff
@@ -1,36 +1,21 @@
+import locators from "../commons/locators";
```

Comment:
Visto que neste arquivo você só está usando os seletores so sub-objeto `LOGIN`, que tal usar desestruturação de objetos, para pegar só estes?
Algo como `import { NOTES } from '../commons/locators'`.
Daí, quando for usar tais _locators_, pode usá-los assim `cy.get(NOTES.LIST_GROUP)`.
E no arquivo de comandos customizados, você poderia fazer `import { LOGIN, NOTES } from '../commons/locators'`.